source/developers/how-tos/maintain-a-repo.rst

```diff
@@ -145,6 +145,77 @@ outlined in :doc:`OEP-54 <openedx-proposals:processes/oep-0054-core-contributors
 
 .. _CCs Wiki Page: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Core+Contributors+to+the+Open+edX+Project
 
+Hold Yourself to Community Standards
+************************************
+
+A prerequisite for a Maintainer is to be a Core Contributor.  Part of what this
+means is that you must hold yourself to the same standards as any CC - any
+contributor, really - in your repository.  You are still a member of the
+community, after all!
+
+Here are a few rules-of-thumb to keep in mind:
+
+* Participate!  Make yourself available in the several community communication
+  channels - Discourse, Slack, working group meetings, and, of course, Github
+  issues - to discuss matters related to your maintenance role publicly.  Don't
+  restrict yourself to private conversation channels: this would make for an
+  opaque project.
+
+* Treat any change you or your organization want to make to the codebase as an
+  open source contribution.  This means that any and all PRs must be properly
+  reviewed by somebody other than the submitter.  And if the change is
+  user-facing, it must necessarily undergo the public `Product Review Process`_.
+
+.. note::
+
+   Adding a feature toggle around a piece of code does not exempt it from being
+   put through the Product Review Process!  Even experiments must be approved
+   beforehand.
```

Comment:
```suggestion
   beforehand; even if a feature is by default not shown, experimental code causes cognitive, testing, and potentially even release-impacting burdens.
```

I don't know if this is the right wording, but I think it's important to be explicit about the "why must this be approved"